build.gradle

```diff
@@ -172,7 +183,17 @@ sonarqube {
     }
 }
 
-configure(allprojects - project(':conductor-grpc')) {
+configure(
+    allprojects
+    - project(':conductor-grpc')
+    - project(':contribs:index:es7-persistence')
+    - project(':contribs:persistence:common-persistence')
+    - project(':contribs:persistence:postgres-persistence')
+    - project(':contribs:task:kafka-publish')
+    - project(':contribs:event-queue:kafka')
+    - project(':contribs:metrics')
+    - project(':contribs:test-util')
+) {
```

Comment:
Skip linter step on the above modules in contribs. It appears to used its own lint version within the contribs subproject previously so running this will re-format everything differently